packages/usa-prose/src/styles/_usa-prose.scss

```diff
@@ -1,12 +1,45 @@
 @use "uswds-core" as *;
+@use "../../../uswds-core/src/styles/placeholders/typography" as *;
 
 .usa-prose {
   @include typeset($theme-prose-font-family);
 
   & > {
-    @include usa-paragraph-style;
-    @include usa-headings-styles;
-    @include usa-list-styles;
-    @include usa-table-styles;
+    p {
+      @extend %usa-prose-p;
+    }
```

Comment:
Copy over structure of [`usa-paragraph-style`](https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/mixins/typography/usa-content-styles.scss#L8-L12) with existing placeholders